internal/service/common/notifier/event.go

```diff
@@ -46,26 +50,45 @@ func processEvent(ctx context.Context, logger *slog.Logger, client *http.Client,
 	logger.Info("processing data change event",
 		"notificationID", event.NotificationID, "sequenceID", event.SequenceID)
 
+	_ = CallUrl(ctx, logger, client, url, event)
+
+	completionChannel <- &SubscriptionJobComplete{
+		subscriptionID: subscriptionID,
+		notificationID: event.NotificationID,
+		sequenceID:     event.SequenceID,
+	}
+}
+
+func CallUrl(ctx context.Context, logger *slog.Logger, client *http.Client, url string, event Notification) error {
```

Comment:
As mentioned in another comment, rather than exposing this and using it directly from places other than directly within the `Notifier` I'd prefer if we align the alarm functionality to how the resource/cluster components work so that the `Notifier` handles the subscriber jobs and sending of the notifications.   One reason for this is that I plan on refactoring this to allow for sending to different types of subscribers based on the callback URL (i.e., special handling for SMO notifications vs internal notifications between our servers that wouldn't require (or support) OAuth tokens).